### PR TITLE
MGMT-761 move deploymnet to assisted-installer-namespace

### PR DESCRIPTION
**Pull request by filanov**
Will be pushed with https://github.com/tsorya/test-infra/pull/5
And after i sync with UI team

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -51,9 +52,12 @@ update: update_revision_file build
 	docker build -f Dockerfile.bm-inventory . -t $(SERVICE)
 	docker push $(SERVICE)
 
-deploy-all: create-build-dir deploy-mariadb deploy-s3 deploy-service
+deploy-all: create-build-dir deploy-namespace deploy-mariadb deploy-s3 deploy-service
 	echo "Deployment done"
 
+deploy-namespace:
```
Comment (ronniel1) [inline Makefile:58]: Doesn't this need to be done before any other deployment is done?

Comment (filanov) [reply Makefile:58]: `deploy-all: create-build-dir deploy-namespace` it is, no?

Comment (ronniel1) [reply Makefile:58]: there is not ordering between the prerequisites of a target
So deploy-mariadb might run before deploy-namespace
In order to set the ordering, you need to make the deploy-namespace a prerequisite of all the targets that depend on it

Comment (filanov) [reply Makefile:58]: i can add `|` maybe? 
https://www.gnu.org/software/make/manual/html_node/Prerequisite-Types.html

Comment (ronniel1) [reply Makefile:58]: The Makefile's way to do this is for each target to specify its dependencies.
In this case:
    deploy-mariadb: deploy namespace
    deploy-s3: deploy-namespace

etc

Comment (filanov) [reply Makefile:58]: ok will change.

**Comment by ronniel1** [comment]
Need to verify with test-infra

**Comment by filanov** [comment]
> Need to verify with test-infra

Already did. will wait untill quay.io return to work then will ask from Jiri to support it as well, test again and push

**Pull request merged**